monitool/Constants/AppAlert.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  AppAlert.swift
+//  monitool
+//
+//  Created by Christianto Budisaputra on 19/08/21.
+//
+
+import Foundation
+
+struct AppAlert {
```

Comment:
ini masih in progress apa gimanaa?

Reply:
ini container aja, anakannya ada di Shared Components